integrationTests/vm/txsFee/utils/utils.go

```diff
@@ -88,6 +89,28 @@ func doDeployInternal(
 	return scAddr, owner
 }
 
+func generateRandomArray(length int) []byte {
+	rand.Seed(time.Now().UnixNano())
```

Comment:
this can elegantly be refactored as:
```GO
buff := make([]byte, length)
_, _ = rand.Read(buff)
```

`rand` package from the crypto lib should be used

Reply:
done